MAINTAINERS.md

```diff
@@ -0,0 +1,165 @@
+# Kubernetes Project Maintainership
+
+Kubernetes is a big project with many contributors, the project organization is
+a crucial task and the project maintainers are the members that are trusted
+with that task, driving the project, improving the health of the project,
+performing quality control and being responsible over one or more components in
+the project. A Kubernetes maintainer is a contributor who has contributed/is
+contributing to the project regularly and a developer that demonstrated good
+technical judgement. The following is a list of requirements to be a Kubernetes
+maintainer.
+
+Before continuing, it should be noted that we value every contributions, however
+small it may be, even a comment on an issue a valuable contribution to us.
+Maintainership requires time commitment and community trust. You don't have to
+be a maintainer to have a meaningful impact on Kubernetes!
+
+## Maintainership Requirements
+
+In order to become a maintainer, a contributor must satisfy all of the
+requirements listed below.
+
+### 1. Enable 2-Factor Account Authentication for GitHub
+
+See GitHub's documentation about [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication/) for details.
+
+### 2. Be an active contributor
+
+Active contributors are considered to be anyone who meets any of the following criteria:
+
+- Sent more than two pull requests (PRs) in the previous one month, or more than
+  20 PRs in the previous year.
+- Filed more than three issues in the previous month, or more than 30 issues in
+  the previous 12 months.
+- Commented on more than pull requests in the previous month, or more than 50
+  pull requests in the previous 12 months.
+- Marked any PR as LGTM in the previous month.
+- Have collaborator permissions in the Kubernetes GitHub organization.
+
+See [Kubernetes Community Expectations](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/community-expectations.md) for more details.
+
+**TODO:** Move the definition of an active contributor to here or remove this
+section and link to expectations.
+
+#### Contribution types
+
+Contributions are not limited to code. Other contributions include PR reviews,
+discussion participations, test ownership, SIG leadership, documentation, etc.
+As we improve our project tooling, we'll be able to measure and analyze other
+types of contributions and take those into account when evaluating a maintainer
+application, however maintainers still need to be familiar with the design and
+implementation of Kubernetes.
+
+### 3. Be experienced in the project
+
+You need to have more than 50 contributions total and project participation for
+more than a year in the Kubernetes organization.
+
+**Note:** A year time requirement will be replaced once the contributor ladders
+are introduced.
+
+### 4. Find 3 senior maintainer sponsors
```

Comment:
@bgrant0607 thanks for detailing the contributor ladder, it's much clearer now! I'll replace the ambiguous "senior maintainer" with project approver